code/game/turfs/space/space.dm

```diff
@@ -23,7 +23,7 @@
 	if(config)
 		if(config.starlight)
 			for(var/turf/simulated/T in RANGE_TURFS(1,src)) //RANGE_TURFS is in code\__HELPERS\game.dm
-				SetLuminosity(3)
+				SetLuminosity(5,1.5)
```

Comment:
So it seems at /tg/ we might decide to have this be 4,1 and not 5,1.5.

See the updated OP for new pictures. (https://github.com/tgstation/-tg-station/pull/12748)

Decide if you want to re-port the new shit.

gif showing all 3 (old, new1, new2):  https://tgstation13.org/msoshit/starlight.gif